source/manual/vpn.html.md

```diff
@@ -27,7 +27,10 @@ If it goes down, these things will happen:
 
 There's a VPN between AWS Production (only) and UKCloud Production which exists
 only as a workaround for routing certain requests from Licensify to Civica, one
-of Licensify's payment gateways.
+of Licensify's payment gateways. This is undesirable and recorded as
```

Comment:
Very nitpicky of me, but the link text could be interpreted as "this takes me to the Tech Debt board, or to a definition of what Tech Debt is", rather than "this takes me to the card for this particular Tech Debt". I think a slight tweak to the link would disambiguate it 👍 

```suggestion
of Licensify's payment gateways. This is undesirable and [recorded as GOV.UK Tech Debt][tech-debt-card].
```